daal4py/README.md

```diff
@@ -0,0 +1,145 @@
+
+## How to create conda environment for benchmarking
+
+`conda create -n skl_bench -c intel python=3.7 daal4py pandas`
+
+##  Algorithms parameters
+
+Benchmarks can be launched for each algorithm separately. The list of supported parameters for each algorithm provided below.
```

Comment:
```suggestion
You can launch benchmarks for each algorithm separately. The tables below list all supported parameters for each algorithm:

- [General](#general)
- [DBSCAN](#dbscan)
- [RandomForestClassifier](#randomforestclassifier)
- [RandomForestRegressor](#randomforestregressor)
- [pairwise_distances](#pairwise_distances)
- [KMeans](#kmeans)
- [KNeighborsClassifier](#kneighborsclassifier)
- [LinearRegression](#linearregression)
- [LogisticRegression](#logisticregression)
- [PCA](#pca)
- [Ridge Regression](#ridge)
- [SVC](#svc)
- [train_test_split](#train_test_split)
```

sidenote: rename the "General" section, e.g. "Common parameter" or something like this